lib/scripts/test.sh

```diff
@@ -5,7 +5,11 @@ CODECRAFTERS_SUBMISSION_DIR=/app
 
 test -d /app-cached && cp -p -R /app-cached/. "$CODECRAFTERS_SUBMISSION_DIR"
 
-if [ -f /codecrafters-precompile.sh ]; then
+if [ -f /.codecrafters/compile.sh ]; then
```

Comment:
```suggestion
if [ -f "${CODECRAFTERS_SUBMISSION_DIR}/.codecrafters/compile.sh" ]; then
```

(Same change below too)

@glokta1 this is pretty basic, you should be testing your changes somehow before opening a PR for review